test/telemetry_poller_SUITE.erl

```diff
@@ -0,0 +1,94 @@
+-module(telemetry_poller_SUITE).
```

Comment:
We need to have a test that asserts that the built-in vm measurements are working as expected. :) Currently we check that they can be defined but we don't check that they are emitting the events properly.

Also, I believe you forgot to delete the main Elixir test file from the repo.